README.md

```diff
@@ -1,68 +1,85 @@
 [![Build Status](https://travis-ci.org/Adyen/adyen-python-api-library.svg?branch=master)](https://travis-ci.org/Adyen/adyen-python-api-library)
 [![Coverage Status](https://coveralls.io/repos/github/Adyen/adyen-python-api-library/badge.svg?branch=master)](https://coveralls.io/github/Adyen/adyen-python-api-library?branch=master)
 
-# Adyen APIs Library for Python
-
-This library simplifies working with Adyen APIs and allows you to integrate Adyen
-payments within any Python application.
-
-## Integration
-The Library supports all APIs under the following services:
-
-* checkout
-* checkout utility
-* payments
-* modifications
-* payouts
-* recurring
-
-## Requirements
-
+This is the officially supported Python library for using Adyen's APIs.
+## Integration 
+The library supports all APIs under the following services:
+ 
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version:  **v67**
+* [Payments API](https://docs.adyen.com/api-explorer/#/Payment/v64/overview): Our classic integration for online payments. Current supported version:  **v64**
+* [Recurring API](https://docs.adyen.com/api-explorer/#/Recurring/v49/overview): Endpoints for managing saved payment details. Current supported version:  **v49**
+* [Payouts API](https://docs.adyen.com/api-explorer/#/Payout/v64/overview): Endpoints for sending funds to your customers. Current supported version:  **v64**
+* [Orders API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/orders): Endpoints for creating and canceling orders. Current supported version:  **v67**
+* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v1/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version:  **v1**
```

Comment:
docs link needs to be updated to v67 as v1 is probably deprecated(endpoint should still work, but let's change it in another story?)
```suggestion
* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version:  **v67**
```